### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by joseheco**
TODO list project:

Make it with:

-HTML
-CSS
-JS

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,28 +1,26 @@
+// eslint-disable-next-line import/no-cycle
+import formTask from './newTasks.js';
+// eslint-disable-next-line import/no-cycle
+import renderTasks from './render.js';
```
Comment (julie-ify) [inline src/index.js:4]: - [x] You have disabled the linter check on your code. Please enable all linter checks on your code to ensure that your code is consistent and clean.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @joseheco,

Good job so far!

## Highlight!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Good code organization and working app ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

- [x] Please let's fix the live demo link you have in the README as it is rendering a 404 error. You can fix this or remove it from the README as it is not yet a must-have but if you wish to have the live demo please make it work 😄

- [x]  Consider adding instructions on how to clone your repository locally and how to run your app locally in the README so that users can know how to get your app running in their local machine.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (joseheco) [reply src/index.js:4]: Thank you Julie for your response all the linter is working as you can see, all the project its working 100%.

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @joseheco, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**